packages/renderer/src/lib/preferences/PreferencesRenderingItem.svelte

```diff
@@ -88,14 +89,16 @@ $: resetToDefault = false;
         record="{recordUI.original}"
         updateResetButtonVisibility="{updateResetButtonVisibility}"
         resetToDefault="{resetToDefault}"
-        enableAutoSave="{true}" />
+        enableAutoSave="{true}"
+        initialValue="{async () => await getInitialValue(recordUI.original)}" />
```

Comment:
I'm not sure to follow why we need to add await here and why it couldn't return the promise ?